docs/api/resource.rst

```diff
@@ -265,21 +265,43 @@ an error response is returned with status ``400``.
 Conflicts
 ---------
 
-Articles URL are unique per user (both ``url`` and ``resolved_url``).
+The Reading List Server provides an automatic conflict resolution
+algorithm for articles.
 
-:note:
-    A ``url`` always resolves towards the same URL. If ``url`` is not unique, then
-    its ``resolved_url`` won't either.
+An article is uniquely defined in the database by its ``url`` and
+``resolved_url`` fields. The ``resolved_url`` field can be different
+or the ``url`` value when the client has to follow one or several
+redirections.
 
 :note:
-    Unicity on URLs is determined the full URL, including location hash.
+    Unicity on URLs is determined by the full URL, including location hash.
     (e.g. http://news.com/day-1.html#paragraph1, http://spa.com/#/content/3)
 
 :note:
     Deleted records are not taken into account for field unicity.
 
-If the a conflict occurs, an error response is returned with status ``409``.
-A ``existing`` attribute in the response gives the offending record.
+
+When a client pushes an new article which ``url`` or ``resolved_url`` already
+exists in the database, the automatic conflict resolver will simply keep
+the original one with all its values (title, summary etc.) and return
+to the client its information. No duplicate is created.
+
+Updating the title, excerpt or word_count of an existing article
+won't raise any conflict: the last call wins.
+
+For both updates and creation, you can bypass the automatic
+conflict resolution by adding a ``If-Unmodified-Since`` in your requests
+headers.
+
+In that case, you will get a 412 error if :
+- you are trying to POST a collection and something was changed in the database
+  in the interim.
+- you are trying to POST an article that was changed in the interim.
```

Comment:
I'm not it's still the case, but we used the passive form in the docs, without using "you" :)